src/specfem3D/write_output_ASDF.f90

```diff
@@ -74,7 +74,7 @@ end subroutine init_asdf_data
 !
 
 !> Stores the records into the ASDF structure
-!! \param asdf_container The ASDF data container
+!! \param asdf_container The ASDF data container 
```

Comment:
I don't think those changes have been included in this PR. The way to do it
would have been to close the PR and start a new one.
Once a PR has been submitted, the changes are frozen. So you should open a
new PR if you want to push your latest changes to the repo.

On Mon, Sep 21, 2015 at 3:34 PM, James Smith <notifications@github.com>
wrote:

> In src/specfem3D/write_output_ASDF.f90
> <https://github.com/geodynamics/specfem3d_globe/pull/436#discussion_r40015060>
> :
>
> > @@ -74,7 +74,7 @@ end subroutine init_asdf_data
> >  !
> >
> >  !> Stores the records into the ASDF structure
> > -!! \param asdf_container The ASDF data container
> > +!! \param asdf_container The ASDF data container
>
> Yeah I just did, thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem3d_globe/pull/436/files#r40015060>.
>



-- 
David Luet
Linux Administrator/Software & Programming Analyst
Department of Geosciences & PICSciE
Princeton University